packages/legend-shared/src/network/NetworkUtils.ts

```diff
@@ -493,6 +496,21 @@ export const downloadFile = (
   }
 };
 
+export const readFileAsText = (file: File): Promise<string> =>
```

Comment:
Why do we throw this function in `NetworkUtils`?

Reply:
I think it made sense before as I had written a more generic read loaded file method. `GeneralUtils` ?

Reply:
I think it does below in `NetworkUtils` because of the File input. 
Docs: /** Provides information about files and allows JavaScript in a web page to access their content. */